wit/world.wit

```diff
@@ -0,0 +1,11 @@
+
+world outbound-keyvalue {
+	import readwrite: pkg.wasi-keyvalue.outbound-keyvalue-readwrite
+	import atomic: pkg.wasi-keyvalue.outbound-keyvalue-atomic
+	import get-many: pkg.wasi-keyvalue.outbound-keyvalue-get-many
+	import set-many: pkg.wasi-keyvalue.outbound-keyvalue-set-many
+}
+
+world inbound-keyvalue {
+	export handle-watch: pkg.wasi-keyvalue.handle-watch
+}
```

Comment:
I would expect that this second world would include a copy of all the contents of the first world, and add this `export`.  I'd also expect to see an imported interface that had a `watch` function to register interest in watching a key?  Or is that handled via external configuration in a more coarse-grained fashion?

Reply:
The `watch` function will be something similar to HTTP `router` func, because it is the one that registers a watch handler function in the host so that the host knows which handler to invoke when the watching key event arrives. 

If we want to go with a more serverless style, I'd say we leave the `watch` to the external configuration.

Reply:
I will close this comment for now as we discussed this in the meeting. There needs a careful design of `handle-watch`